eth/protocols/eth/protocol.go

```diff
@@ -30,9 +30,9 @@ import (
 
 // Constants to match up protocol versions and messages
 const (
-	ETH66 = 66
-	ETH67 = 67
-	ETH68 = 68
+	ETH66 = iota + 66
+	ETH67
+	ETH68
```

Comment:
iota is useful in cases of "we don't care about the actual numbers, let the compiler figure it: all we require is a sequence of unique and increasing numbers". 

For protocol definitions, we _do_ care about what the actual numbers are, since they are required to be a certain number in order for communication with peers to work. If we use iota and then drop `ETH67` for some reason, then suddenly `ETH68` becomes `67`. oops.